### ISSUE TITLE
Implement new design for backoffice

### ISSUE THREAD

**Issue by seballot** [openfoodfoundation/openfoodnetwork#8525]
This new design has been approved and needs to be implemented

https://www.figma.com/file/UOUOI0AdaL3gRmZWacHnsO/OFN-Color-Choice?node-id=2%3A627

I will start working on it soon

This design include ability to select multiple products and do bulk actions, but this should be a dedicated feature (or maybe it's a pending PR?)

**Comment by jibees** [comment]
> This design include ability to select multiple products and do bulk actions, but this should be a dedicated feature (or maybe it's a pending PR?)

It's part of the Backoffice UI Uplift, which is our next big thing: https://github.com/openfoodfoundation/openfoodnetwork/issues/7198

A single PR, with _only_ the refresh of the design would be awesome! Feel free to ask for anything. 🙏

**Comment by jibees** [comment]
Recently, we wanted to introduce tailwind as a new CSS _framework_
The [PR#8134](https://github.com/openfoodfoundation/openfoodnetwork/pull/8134) is still in draft due to too many priorities. Wondering if it's not worthing to awake this PR before starting such an issue...

**Comment by seballot** [comment]
Hi @jibees, I read some of your past disucssion re Css framework. I must confess I'm a bit skeptical about using tailwind :)

I saw that rails ViewComponent have been introduced recently in OFN
Stimulus also have been introduced recently
Angular is also used in the project
coffeScript & Haml are also used. Kind of standard for Rails, but not in all cases (I never used coffee script for example)
Now you want to introduce a Css framework, which does not seems straighforward to install when I see how you are struggling to add it to OFN stack, and which is bit popular, but not commonly used by rails community

Each step can be a potential barrier to a new dev in the project

What is the plan with tailwind? 
1. rewrite all scss? -> I don't think so, too much work !!
2. use it only fo new views, and use now 100% tailwind, no more standard css? -> this will need all developpers to be trained to tailwind, so could be a barrier
3. just use it here and there when it's needed, like for adding some padding or so, or one off components, but still use standard css classes like `.btn` that will be defined with scss ? -> if so, then why not just use bootstrap5 utilities? it comes as a separate scss file that we can use without the whole bootstrap. Just need to install bootstrap5 package and import utilities in application.scss, that's it. it's a bit similar to tailwind, but I think that more people know bootstrap that tailwind

Sorry, I'm just arriving in the project, of course I didn't followed all discussion and so on... it was just my two cents contribution to the discussion

Depending on what you want to do with tailwind (1,2,3), it might not be necessary to wait for it in order to start reshaping back office, but I will let you decide, I can do other stuff in the meantime. Let me know !

See ya !

**Comment by jibees** [comment]
> Now you want to introduce a Css framework, which does not seems straighforward to install when I see how you are struggling to add it to OFN stack, and which is bit popular, but not commonly used by rails community

I think you reach the point here. 
Two (or already 3?) months ago, we chose to implement tailwind in the OFN, and (because of many discussions appears and the whole team wasn't exactly aligned) we decided that we must re-considered its introduction and see if it's again aligned with what we are building here at OFN.
I was a big fan of tailwind (and I'm still fan actually), but I must admit that I probably should re-consider my previous position: I did not anticipate well that it can be very difficult to integrate and use a tool that has not been designed and developed for your existing stack (here RoR with old Angularjs).
So, my position is now to re-engage a conversation with the @openfoodfoundation/core-devs team, and see how we can easily and effectively continue to build, develop and improve an app taking into account its history and the whole ecosystem it belongs to.

(sorry for the 💥 post)

**Comment by seballot** [comment]
Thanks for clarifying this !
It's always hard to manage this kind of apps, which are quite old, have a long history, have been built my multiples people, and the stack it definitly never using the last techs :) I'm facing same situation in various project ! 

To come back to this ticket, maybe I can start a first work consisting of changing the colors to make it more readable (more constrast) ? It will not engage too much changes

And btw @jibees, if you find some time, could you please have a look to the issues I have created in the wishlist project? https://github.com/openfoodfoundation/wishlist/issues Nothing urgent, just to let you know in case you don't have notifications for this repo. Thanks !

**Comment by mkllnk** [comment]
@seballot What would be your preference? Use Bootstrap or build everything from scratch / update current?

**Comment by seballot** [comment]
Hi @mkllnk !

As the project is already mature, I don't think there is a point of including the whole bootstrap5 library : the OFN front end is already nicely structured (i.e every input is encapsulated in a `.field` container etc...) so it's easy to make changes, and some components have already been build (like ofn-dropdown). 

I think bootstrap5 utilities could be nice to have, but I can definitly live without it !

**Comment by jibees** [comment]
@seballot the wishlist is managed by the @openfoodfoundation/train-drivers-product-owners and will certainly have a look soon.
